### PR TITLE
Created a phpbeautifyrc.json file - Solution to issue #2

### PR DESCRIPTION
**Pull request by Shankhanil**
Here are the details of the changes that I have made. 

1. I have created the phpbeautifyrc.json file. I have given default value to the parameters
2. I have inserted a small code piece in /*...*/ comments. The code piece is supposed to read the phpbeautifyrc.json and get the parameters for formatting. But, in my environment it is malfunctioning. I would like _you_ to review it, and suggest the changes. 
3. I would also need to modify the .replace(...) statements in format() in php.js file, and include conditional replacements based on the parameters in .json file. But this could be done only _after the issue in point 2 is resolved_.

### CODE REVIEW

**Comment by nadchif** [comment]
@Shankhanil Thank you once again! I've reviewed it an added a suggested implementation. Check it out and tell me what you think. Remember to run `git pull` locally to pull in the latest changes I've committed

**Review by nadchif — CHANGES_REQUESTED** [review]
Please checkout the suggested implementation and tell me what you think

**Comment by Shankhanil** [comment]
hey @nadchif Great piece of code on your side!. But, I am git push-ing my changes, but they aren't showing up here. 

Also, send me tutorials on how to read preferences from a promise object.

**Review by Shankhanil — COMMENTED** [review]
I have accepted and included the changes in the code. Thanks for the contribution.

File: src/thirdparty/php.js [hunk 1]
```diff
@@ -66,7 +66,15 @@ define(function(require, exports) {
   const format = (code, customConfiguration) => {
     // use custom configuration here. if there was none found, customConfiguration will still be null
     console.log('Using the following configuration', customConfiguration);
-
+
```
Comment (nadchif) [inline src/thirdparty/php.js]: I already did the piece of code to read the preferences and handle the promises logic. The variable `customConfiguratuon` already loads the preferences. You could use that variable.

To test it, open a folder (project) in brackets that contains a `.phpbeautifyrc.json`, you will see the setting will be loaded and logged into the console by line 68

**Comment by Shankhanil** [comment]
then the PR is ready to be merged.

**Comment by nadchif** [comment]
I've merged the PR into the [development branch](https://github.com/nadchif/brackets-php-beautify/tree/development). We will merge into the [master](https://github.com/nadchif/brackets-php-beautify/tree/master) when the implementation is complete and ready for new version release

**Comment by nadchif** [comment]
Please proceed with the implementation. See line 56 of [src/thirdparty/php.js](https://github.com/nadchif/brackets-php-beautify/blob/development/src/thirdparty/php.js) on the [development](https://github.com/nadchif/brackets-php-beautify/tree/development) branch

**Comment by Shankhanil** [comment]
I will look into it ASAP!

**Pull request merged**